website/static/js/pages/project-dashboard-page.js

```diff
@@ -122,9 +122,8 @@ $(document).ready(function () {
         currentUserCanEdit: window.contextVars.currentUser.canEdit
     });
     var newComponentElem = document.getElementById('newComponent');
-    if (newComponentElem) {
-        m.mount(newComponentElem, AddComponentButton);
-    }
+
+    m.startComputation();
 
     if (ctx.node.institutions.length && !ctx.node.anonymous && !ctx.node.isRetracted) {
```

Comment:
I think this mount can be moved inside the `if (!ctx.node.isRetracted) {` block (and remove the isRetracted check)